android/src/main/java/com/tailscale/ipn/MainActivity.kt

```diff
@@ -106,7 +105,7 @@ class MainActivity : ComponentActivity() {
 
     setContent {
       AppTheme {
-        val navController = rememberNavController()
+        navController = rememberNavController()
```

Comment:
This is a drive-by fix to avoid errors like 

```
kotlin.UninitializedPropertyAccessException: lateinit property navController has not been initialized
                                                                                                    	at com.tailscale.ipn.MainActivity.onNewIntent(MainActivity.kt:248)                                                                                               	
```